pkg/config/plugins/policies/policies.go

```diff
@@ -0,0 +1,22 @@
+package policies
+
+func DefaultPoliciesConfig() *Config {
+	return &Config{
+		PluginPoliciesEnabled: []string{
+			"meshaccesslogs",
```

Comment:
Is it better to make these strings constants?

Reply:
not sure it matters that much, I already have this in this format and it's autogenerated so would require more changes. If you really want to change it please open up an issue.